src/main/resources/application.yaml

```diff
@@ -221,12 +221,15 @@ govnotify:
         legalRep:
           email: e79249e2-ecbf-4d2b-8458-006285613402
     caseEdited:
+      homeOffice:
+        email: b4cfdb17-6130-4f52-aec4-f3571e030790
```

Comment:
@burale there is a new template here but where is being used / called ?

Reply:
This is not new template and it is used in "HomeOfficeEditListingPersonalisation" class.